src/tritongrader/formatter.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         max_output_bytes: int = 5000,
         verbose: bool = True,
         html_diff: bool = True,
+        leaderboard: List[Tuple[str, Optional[Union[int, float, str]], Optional[str]]] = None,
```

Comment:
How does this on gradescope? Do we need our own backend to generate the ranking or do we only need to emit one record per submission?

Reply:
We only need to add a record to each results.json
[https://gradescope-autograders.readthedocs.io/en/latest/leaderboards/](https://gradescope-autograders.readthedocs.io/en/latest/leaderboards/)
Each metric should contain name, value(optional), order(optional)
Gradescope will automatically handle rankings based on order

Reply:
I see. That sounds like something that could be implemented into this library. I'll think about it and make some plans when I have time. Or feel free to propose changes.